src/coreclr/jit/codegenarm64.cpp

```diff
@@ -10140,4 +10152,46 @@ void CodeGen::genCodeForAddEx(GenTreeOp* tree)
     genProduceReg(tree);
 }
 
+//------------------------------------------------------------------------
+// genCodeForCond: Generates the code sequence for a GenTree node that
+// represents a conditional instruction.
+//
+// Arguments:
+//    tree - conditional op
+//
+void CodeGen::genCodeForCond(GenTreeOp* tree)
+{
+    assert(tree->OperIs(GT_CSNEG_MI));
+    assert(!(tree->gtFlags & GTF_SET_FLAGS) && (tree->gtFlags & GTF_USE_FLAGS));
```

Comment:
nit: Might be nice to have a `GenTree::gtUseFlags()` function to parity the `gtSetFlags()` one

Reply:
See my comment about `gtSetFlags` and `gtRequestSetFlags`.